### PR TITLE
redcap is_complete: return None if {instrument}_complete does not exist

### PR DESCRIPTION
**Pull request by joverlee521**
Extend encapsulation by supporting three-valued logic. Makes it easier
for us to differentiate between an incomplete instrument and the
instrument not existing in data.

### CODE REVIEW

**Comment by joverlee521** [comment]
[Update `is_complete()` to return `None` if instrument does not exist](https://trello.com/c/WCsPqnXD/693-update-iscomplete-to-return-none-if-instrument-does-not-exist)

File: lib/id3c/cli/redcap.py [hunk 1]
```diff
@@ -255,10 +258,15 @@ def is_complete(instrument: str, data: dict) -> bool:
     True
     >>> is_complete("test", {"test_complete": "Incomplete"})
     False
-    >>> is_complete("test", {})
-    False
+    >>> is_complete("test", {}) is None
+    True
```
Comment (jccraft-uw) [inline lib/id3c/cli/redcap.py:262]: The is_complete("test", {}) case will return None or True? Is the "True" dangling here?

Comment (joverlee521) [reply lib/id3c/cli/redcap.py:262]: `is_complete("test", {})` should return `None`. This is checking `is_complete("test", {}) is None`, which should return `True`. 

There might be a better way of writing this particular test?

Comment (jccraft-uw) [reply lib/id3c/cli/redcap.py:262]: That makes sense!

**Pull request merged**